provision/provision-helpers.sh

```diff
@@ -17,11 +17,18 @@ export BLUE="\033[0;38;5;4m" # 33m"
 export PURPLE="\033[0;38;5;5m" # 129m"
 export CRESET="\033[0m"
 
-
-VVV_CONFIG=/vagrant/vvv-custom.yml
-if [[ -f /vagrant/config.yml ]]; then
-	VVV_CONFIG=/vagrant/config.yml
-fi
+configs=(
```

Comment:
Should we turn this snippet into a function (including the export below the change here)?

This would reduce some code duplication, and potentially improve maintainability (same code is implemented in the test suite on provision/tests/provisioners.sh )

Reply:
I'd like that but we'll need this in utilities etc for a while eitherway

Reply:
This file we're commenting on is already on provision helpers. Utilities can have their own implementations that would override this exported value.

On that topic, at some point we'd need to have some sort of versioning for easier introduction of breaking changes to utilities. Eg: utilities v2, only works with VVV v3.6, or something on those lines. Might be troublesome to implement, but will allow faster development on the utilities repo.

Reply:
I think a polyfill script is probably best for that, and a minimum VVV version helper function

Reply:
Maybe we can open a ticket for that and move on this pr?